include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl.h

```diff
@@ -847,7 +858,7 @@ struct __get_zeroth_element
         return std::get<0>(std::forward<_Tp>(__a));
     }
 };
-
+template <int32_t __offset = 0, typename Assign = oneapi::dpl::__internal::__pstl_assign>
```

Comment:
Is there a reason we use a signed type here?

Reply:
It provides it with a little more utility in the future to do other things.  Its very unlikely we would use a negative value here though.  The only options right now are `0` or `1`, but I think we don't lose anything to allow it to be signed. I think it will be combined at compile time with the `- 1` which precedes it.

Reply:
No objections from my side. I guess we should just use `std::int32_t` to align with our convention.

Reply:
fixed.